macOS CI does not properly fail in preparation step if ffmpeg failed to be installed

It fails during the building step instead.

Output of `Prepare macOS` step in https://github.com/ddnet/ddnet/actions/runs/17729306995/job/50376853737#step:6:152

```
==> Fetching libvidstab
==> Downloading https://ghcr.io/v2/homebrew/core/libvidstab/blobs/sha256:f76b62fb9580b75861cc5a6d639500d095823d32bef7aa6f636fee2d87b724e9
curl: (56) The requested URL returned error: 502
Error: ffmpeg: Failed to download resource "libvidstab"
Download failed: https://ghcr.io/v2/homebrew/core/libvidstab/blobs/sha256:f76b62fb9580b75861cc5a6d639500d095823d32bef7aa6f636fee2d87b724e9
```

Output of `Build in debug mode` step in https://github.com/ddnet/ddnet/actions/runs/17729306995/job/50376853737#step:11:206

```
-- Configuring done (11.0s)
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
/Users/runner/work/ddnet/ddnet/AVCODEC_INCLUDEDIR
   used as include directory in directory /Users/runner/work/ddnet/ddnet
/Users/runner/work/ddnet/ddnet/AVFORMAT_INCLUDEDIR
   used as include directory in directory /Users/runner/work/ddnet/ddnet
/Users/runner/work/ddnet/ddnet/AVUTIL_INCLUDEDIR
   used as include directory in directory /Users/runner/work/ddnet/ddnet
/Users/runner/work/ddnet/ddnet/SWRESAMPLE_INCLUDEDIR
   used as include directory in directory /Users/runner/work/ddnet/ddnet
/Users/runner/work/ddnet/ddnet/SWSCALE_INCLUDEDIR
   used as include directory in directory /Users/runner/work/ddnet/ddnet
AVCODEC_LIBRARY (ADVANCED)
    linked by target "game-client" in directory /Users/runner/work/ddnet/ddnet
AVFORMAT_LIBRARY (ADVANCED)
    linked by target "game-client" in directory /Users/runner/work/ddnet/ddnet
AVUTIL_LIBRARY (ADVANCED)
    linked by target "game-client" in directory /Users/runner/work/ddnet/ddnet
SWRESAMPLE_LIBRARY (ADVANCED)
    linked by target "game-client" in directory /Users/runner/work/ddnet/ddnet
SWSCALE_LIBRARY (ADVANCED)
    linked by target "game-client" in directory /Users/runner/work/ddnet/ddnet

CMake Generate step failed.  Build files cannot be regenerated correctly.
-- Generating done (0.1s)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -102,8 +102,8 @@ jobs:
     - name: Prepare macOS
       if: contains(matrix.os, 'macOS')
       run: |
-        brew update || true
-        brew install pkg-config sdl2 ffmpeg ninja molten-vk rust || true
+        brew update
+        brew install ffmpeg molten-vk ninja pkg-config rust sdl2
         brew upgrade freetype
         pip3 install --break-system-packages dmgbuild
         echo /Library/Frameworks/Python.framework/Versions/3.12/bin >> $GITHUB_PATH

